packages/svelte-docgen/src/doc/type.ts

```diff
@@ -33,29 +33,36 @@ export type Type =
 export type TypeOrRef = Type | TypeRef;
 
 /**
- * Type where `alias` is _optional_.
+ * Type that can optionally have an `alias`
  */
 export interface WithAlias {
 	alias?: string;
 	/** Alias type arguments */
 	aliasTypeArgs?: TypeOrRef[];
 	/**
-	 * Where is this type declared?
+	 * Where is this type alias declared?
 	 */
-	sources?: Set<string>;
+	aliasSource?: string;
```

Comment:
Do I understand correctly that each alias for the same type can have only _one_ source?

Reply:
AFAIK, yes, that’s correct.

A type alias can only originate from a single source. In contrast, an interface can be merged from multiple sources, which is one of the key features of TypeScript’s interfaces.